test/functional/mempool_limit.py

```diff
@@ -20,55 +23,76 @@ def set_test_params(self):
         ]]
         self.supports_cli = False
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
         txouts = gen_return_txouts()
-        relayfee = self.nodes[0].getnetworkinfo()['relayfee']
+        # Helper function that generates large transactions
+
+        node = self.nodes[0]
+        miniwallet = MiniWallet(node)
+        relayfee = node.getnetworkinfo()['relayfee']
 
         self.log.info('Check that mempoolminfee is minrelytxfee')
```

Comment:
while you're here, you can fix the typo 
```suggestion
        self.log.info('Check that mempoolminfee is minrelaytxfee')
```